### DISCUSSION TITLE
File signature does not match ROFL format

### DISCUSSION THREAD

**Discussion by MegicArt** [fraxiinus/ReplayBook#303] [Troubleshooting]
Just made a new replay. Added a folder with this replay to a program. In the program I have this error:
![image](https://github.com/fraxiinus/ReplayBook/assets/89039874/d4daedcf-7255-49fb-8d1a-03426102d337)
I tried to re-download a replay but nothing changed.
Can someone help me please

**Comment by fraxiinus** [comment]
Known issue, see pinned message on website or README:

#299
